src/pretix/plugins/sendmail/views.py

```diff
@@ -29,6 +29,12 @@ class SenderView(EventPermissionRequiredMixin, FormView):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['event'] = self.request.event
+        from_log_id = self.request.GET.get('from_log')
+        message = LogEntry.objects.get(id=from_log_id).display()
```

Comment:
This will cause an error whenever you don't have the ``from_log`` parameter. I would enclose all of this in a ``if 'from_log' in self.request.GET`` block.